web/public/index.html

```diff
@@ -26,10 +26,32 @@
     -->
     <title>Tracetest</title>
     <script>
-      window.ENV = {
-        measurementId: "{{ .MeasurementId }}",
-        analyticsEnabled: "{{ .AnalyticsEnabled }}"
+      function getTemplateValue(value, defaultValue) {
+        // hack: we cannot add double curly brackets on our code otherwise the go template
+        // module will try to parse it and it will fail.
+        var curlyBracket = String.fromCharCode(123);
+        var startingTemplateToken = `${curlyBracket}${curlyBracket}`;
+        if (!value || value.startsWith(startingTemplateToken)) {
+          return defaultValue
+        }
+
+        return value
+      }
```

Comment:
For some reason, when this file is built, the JS code is simplified by the minifier. So, I had to add this dirty hack to be able to detect variables containing template formatting.